bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBigtableDataClient.java

```diff
@@ -152,6 +152,16 @@
    *
    * @param request a {@link Query} object.
    * @param observer a {@link StreamObserver} object.
+   * @deprecated Please use {@link #readRowsAsync(Query, StreamObserver)}.
    */
+  @Deprecated
   void readFlatRowsAsync(Query request, StreamObserver<FlatRow> observer);
+
+  /**
+   * Reads {@link Row} asynchronously, and pass them to a stream observer for processing.
+   *
```

Comment:
Reads {@link Row} asynchronously, and pass them to a stream observer for processing.
-->
Reads rows asynchronously and passes them to the given stream observer for processing.

Reply:
Thanks a lot the suggestion, Have updated JavaDoc with the same.